repos/system_upgrade/el7toel8/actors/yumupgradability/actor.py

```diff
@@ -0,0 +1,43 @@
+from leapp.actors import Actor
+from leapp.models import Inhibitor
+from leapp.tags import IPUWorkflowTag, ExperimentalTag, InitRamStartPhaseTag
+from leapp.libraries.stdlib import call
+
+
+class YumUpgradability(Actor):
+    """
+    An actor that takes care of yum upgradability.
+    """
+
+    name = 'yum_upgradability'
+    consumes = ()
+    produces = (Inhibitor,)
+    tags = (IPUWorkflowTag, InitRamStartPhaseTag, ExperimentalTag)
+
+    def secure_yum_upgradability(self):
```

Comment:
As we discussed with @vinzenz, the best would be move of this function into the tool - means, create sh script that will do these 3 operations. Additionally, it should be tool usable by any actor in the repository, so it should be under `repos/system_upgrade/el7toel8/tools` directory (@vinzenz  correct me in case I missed something)

Reply:
@pirat89 I've received @vinzenz 's confirmation for `repos/system_upgrade/el7toel8/tools`.